cli/src/providers/aws/APIProxy.py

```diff
@@ -59,6 +59,9 @@ def get_ips_for_feature(self, component_key):
         result.sort()
         return result
 
+    def login(self, env=None):
```

Comment:
Maybe it will be good to add some comment there and say why we have done this that way?